ament_flake8/ament_flake8/main.py

```diff
@@ -139,8 +139,10 @@ def generate_flake8_report(config_file, paths, excludes, max_line_length=None):
         return generate_flake8_report(config_file, paths, excludes, max_line_length)
 
     flake8_argv = []
-    flake8_argv.append('--config={0}'.format(config_file))
-    flake8_argv.append('--exclude={0}'.format(excludes))
+    if config_file is not None:
+        flake8_argv.append('--config={0}'.format(config_file))
+    if excludes is not None:
```

Comment:
That is not possible. `argparse` won't allow you to do that with the current settings.

Also in that case I think the code you commented on is doing exactly the right think: `excludes is not None` and therefore calls the script with `--excluded=`.